test.js

```diff
@@ -0,0 +1,43 @@
+// Take all files in [folderA] and run them against schema in [folderA] named [fileA].schema.json, or [default]
+
+const fs = require('fs');
+var exec = require('child_process').exec;
+
+const DEFAULT_SCHEMA = 'DCC.ValueSets.schema.json';
+const AJV_COMMAND = './node_modules/.bin/ajv  -c ajv-formats --spec=draft2020 --strict=false test';
+const EXCLUDES = [
+    'package.json',
+    'package-lock.json'
+];
+
+// Get a list of all of the valuesets
+valuesets = fs
+    .readdirSync('./', {withFileTypes: true})
+    // Exclude directories
+    .filter(item => !item.isDirectory())
+    // Exclude package +
+    .filter(item => !EXCLUDES.find(x => x == item.name))
+    // Only include JSON files
+    .filter(item => item.name.endsWith(".json"))
+    .map(item => item.name);
+
+// Get a list of all of the schemas
+schemas = fs
+    .readdirSync('./schemas/', {withFileTypes: true})
+    .filter(item => !item.isDirectory())
+    .filter(item => item.name.endsWith(".json"))
+    .map(item => item.name);
+
+valuesets.forEach(item => {
+    // Decide which schema to use - the named schema or the default
+    let schema = schemas.find(x => x === item.replace(".json", ".schema.json")) || DEFAULT_SCHEMA;
+    // Build up the CLI command
+    let command = `${AJV_COMMAND} -s "./schemas/${schema}" -d "${item}" --valid`;
+    exec(command,
+        function (error, stdout, stderr) {
+            console.log(stdout);
+            if (error !== null) {
+                console.log('stderr: ' + stderr);
```

Comment:
In case of failure, it'd make sense to `process.exit(1)` so that the GitHub Action fails - I don't think it does just from printing something to `stderr`. (Or does it?) (For the user it'd be most convenient if *all* value sets get reported on, not only the first one that fails.)

Reply:
I checked: even with an error on `stderr`, the GitHub Action doesn't fail.

Reply:
Thought so, I've always had to use an explicit exit code (works the same on Windows, it gets swallowed).